README.md

```diff
@@ -33,7 +33,19 @@ go run cmd/siglens/main.go --config server.yaml
 `TBD`
 
 ### Using SigLens Docker
-`TBD`
+The SigLens backend is deployed independently of the UI. 
+To allow the UI to connect to the backend a docker network can be used.
+```bash
+wget https://sigscalr-configs.s3.amazonaws.com/0.1.0 /server.yaml
```

Comment:
We should try to package the server.yaml also as part of the docker ? (Or keep this file outside of s3)

Reply:
we can upload assets with github releases, will change this once we have that workflow ready